### PR TITLE
swap to standard linter

### PR DESCRIPTION
**Pull request by wildan3105**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,22 +10,28 @@
...
+    "eslint-plugin-import": "^0.1.0",
     "expect": "^1.20.2",
     "localtunnel": "^1.8.3",
     "mocha": "^3.4.2",
+    "standard": "^10.0.3",
     "xo": "^0.18.2"
   },
   "xo": {
-    "space": 4
+    "standard": "^10.0.3"
```
Comment (hiimbex) [inline package.json]: this whole `xo` object should be removed

File: package.json [hunk 2]
```diff
@@ -10,22 +10,28 @@
     "xmlhttprequest": "^1.8.0"
   },
   "devDependencies": {
-    "eslint-config-probot": "^0.1.0",
+    "eslint-plugin-import": "^0.1.0",
     "expect": "^1.20.2",
     "localtunnel": "^1.8.3",
     "mocha": "^3.4.2",
+    "standard": "^10.0.3",
     "xo": "^0.18.2"
```
Comment (hiimbex) [inline package.json]: `xo` should be removed here as well

File: package.json [hunk 3]
```diff
@@ -10,22 +10,28 @@
     "xmlhttprequest": "^1.8.0"
   },
   "devDependencies": {
-    "eslint-config-probot": "^0.1.0",
+    "eslint-plugin-import": "^0.1.0",
```
Comment (hiimbex) [inline package.json]: please run `npm install --save-dev eslint-plugin-import` to correctly install this line

File: package.json (hunk 2 above)
Comment (wildan3105) [reply package.json]: okay, will do that immediately!

**Comment by wildan3105** [comment]
Edited it.

**Pull request merged**